Sources/GuakaClILib/MiscTypes.swift

```diff
@@ -57,8 +77,10 @@ public enum GuakaError: Error {
         "   guaka new my-command",
         "   guaka new my_command",
         "   guaka new myCommand"].joined(separator: "\n")
+
     case .tooManyArgsPassed:
-      return "Too many arguments passed to command."
+      return "Too many arguments passed to command.".f.red
```

Comment:
Can't we just ignore the extra args? Also we could say something like "Too many arguments. Trailing arguments will be ignored"